### PR TITLE
Fix dev tools and add some new, Criteo specific ones.

### PR DESCRIPTION
**Pull request by AnthonyTruchet**

### CODE REVIEW

File: dev/criteo/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+### Local compilation and testing ###
+
+* criteo-spark-env.sh: to be *sourc'ed* by other scrip to provide consistent configuration
+* build-with-maven.sh: package Spark using *the right* (really?) options for the Criteo build of Spark. 
+  Additional argument can be passed, they will be forwarded to maven. Tests **are skipped**.
+* test-with-maven.sh: launch Spark unit testing with options for the Criteo build of Spark. 
+  Additional argument can be passed, they will be forwarded to maven. Most useful one are
+  
+  ```
+  ./dev/criteo/build-with-maven -pl mllib # Run tesst only for one maven module
```
Comment (superbobry) [inline dev/criteo/README.md:15]: Typo: tesst.

File: dev/criteo/README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+## Criteo specific developer's scripts ##
+
+This directory holds a bunch of script intended for contributing to Spark from the Criteo environment. 
+The content of this directory *is not to be contributed upstream*.
+
+### Local compilation and testing ###
+
+* criteo-spark-env.sh: to be *sourc'ed* by other scrip to provide consistent configuration
```
Comment (superbobry) [inline dev/criteo/README.md:8]: Typo: scrip.

**Review by ashangit — APPROVED** [review]

File: dev/criteo/README.md (hunk 2 above)
Comment (AnthonyTruchet) [reply dev/criteo/README.md:8]: Done in new PR #20 (this one is already merged)

File: dev/criteo/README.md (hunk 1 above)
Comment (AnthonyTruchet) [reply dev/criteo/README.md:15]: Done in new PR #20 (this one is already merged)

**Pull request merged**